### PR TITLE
#417 - Parsing scope from string

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #417 
Parsing scope from string

### CODE REVIEW

File: src/test/java/com/artipie/docker/http/ScopeTest.java [hunk 1]
```diff
@@ -50,4 +50,24 @@ void pushScope() {
             new IsEqual<>("repository:busybox:push")
         );
     }
+
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/docker/http/ScopeTest.java:53]: @olegmoz I'd suggest to add test for the case when `source` is not valid

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@olegmoz one suggestion, take a look please

**Comment by olegmoz** [comment]
@olenagerasimova thanks, added negative test cases. Please review

**Comment by olenagerasimova** [comment]
@olegmoz are you sure you've pushed the changes? I see only merge from master

**Comment by olegmoz** [comment]
@olenagerasimova sorry, I really did not push the changes

**Review by olenagerasimova — APPROVED** [review]
@olegmoz thanks, looks good to merge

**Pull request merged**